configurations/coding_plans.py

```diff
@@ -123,6 +123,7 @@ def clean_age_with_range_filter(text):
     CodingPlan( raw_field="s01_lessons_learnt_raw",
                 dataset_name="s01_lessons_learnt",
                 time_field="sent_on",
+                run_id_field="rqa_lessons_learnt_run_id",
```

Comment:
```suggestion
                run_id_field="lessons_learnt_run_id",
```
Because even though we are treating it as an radio question its not really a radio question

Reply:
Thanks, updated